storybook/pages/AdvancedSettingsPage.qml

```diff
@@ -0,0 +1,267 @@
+                fleet: mockData.fleet
+                bloomLevel: mockData.bloomLevel
+                wakuV2LightClientEnabled: mockData.wakuV2LightClientEnabled
+                isTelemetryEnabled: mockData.isTelemetryEnabled
+                isAutoMessageEnabled: mockData.isAutoMessageEnabled
+                isDebugEnabled: mockData.isDebugEnabled
+                isCommunityHistoryArchiveSupportEnabled: mockData.isCommunityHistoryArchiveSupportEnabled
+
+                // customNetworksModel: advancedModule? advancedModule.customNetworksModel : []
+                customNetworksModel: []
+
+                isWakuV2: true
+
+                function toggleExperimentalFeature(feature) {
+                    logs.logEvent("advancedStore::toggleExperimentalFeature", ["feature"], arguments)
+                    if (feature === "wallet") {
+                        mockData.accountSettings.isWalletEnabled = !mockData.accountSettings.isWalletEnabled
+                    }
+                    if (feature === "browser") {
+                        mockData.accountSettings.isBrowserEnabled = !mockData.accountSettings.isBrowserEnabled
+                    }
+                    if (feature === "communityHistoryArchiveSupport") {
+                        mockData.isCommunityHistoryArchiveSupportEnabled = !mockData.isCommunityHistoryArchiveSupportEnabled
+                    }
+                    if (feature === "nodeManagement") {
+                        mockData.accountSettings.nodeManagementEnabled = !mockData.accountSettings.nodeManagementEnabled
+                    }
+                    if (feature === "communityPermissions") {
+                        mockData.accountSettings.isCommunityPermissionsEnabled = !mockData.accountSettings.isCommunityPermissionsEnabled
+                    }
+                    if (feature === "discordImportTool") {
+                        mockData.accountSettings.isDiscordImportToolEnabled = !mockData.accountSettings.isDiscordImportToolEnabled
+                    }
+                }
+
+                function setWakuV2LightClientEnabled(mode) {
+                    logs.logEvent("advancedStore::setWakuV2LightClientEnabled", ["mode"], arguments)
+                    mockData.wakuV2LightClientEnabled = mode
+                }
+                function setBloomLevel(mode) {
+                    logs.logEvent("advancedStore::setBloomLevel", ["mode"], arguments)
+                    mockData.bloomLevel = mode
+                }
+                function toggleDebug() {
+                    logs.logEvent("advancedStore::toggleDebug")
+                    mockData.isDebugEnabled = !mockData.isDebugEnabled
+                }
+                function toggleAutoMessage() {
+                    logs.logEvent("advancedStore::toggleAutoMessage")
+                    mockData.isAutoMessageEnabled = !mockData.isAutoMessageEnabled
+                }
+                function toggleTelemetry() {
```

Comment:
Something is wrong when trying to switch `telemetry` option:
"QML ConfirmationDialog: cannot find any window to open popup in."